client/vue.config.js

```diff
@@ -12,14 +12,6 @@ module.exports = {
   publicPath: process.env.VUE_APP_STATIC_PATH,
   chainWebpack: config => {
     config.resolve.symlinks(false);
-    config.module
-      .rule("additional")
-      .test(/\.m?jsx?$/)
-      .include.add(/@girder\/components/)
-      .add(/vue-utilities/)
-      .end()
-      .use()
-      .loader("babel-loader");
```

Comment:
This broke my build.  I have no idea what it was here for.  I definitely don't know why JSX is being referenced.